python/tk_framework_desktopserver/server.py

```diff
@@ -21,13 +21,15 @@
 
 from .errors import MissingCertificateError, PortBusyError
 
+from .logger import get_logger
+
 
 class Server:
     _DEFAULT_PORT = 9000
     _DEFAULT_KEYS_PATH = "../resources/keys"
     _DEFAULT_WHITELIST = "*.shotgunstudio.com"
 
-    def __init__(self, port=None, debug=True, whitelist=None, keys_path=None):
+    def __init__(self, port=None, debug=False, whitelist=None, keys_path=None):
```

Comment:
Nah, we should do it as we find them. As I said the other day, the framework needs a lot of love documentation wise so we'd better start now.